src/ol/Overlay.js

```diff
@@ -120,7 +120,9 @@ class Overlay extends BaseObject {
    * @param {Options} options Overlay options.
    */
   constructor(options) {
-    super();
+    super({
+      [Property.MAP]: null,
+    });
```

Comment:
Is this the best way to initialize it to null?

Reply:
I don't think this would help, because tsc wouldn't know what to do with it. Better change the `getMap()` method to `return this.get(Property.MAP) || null;`